### PR TITLE
CMP-2101: SPO 0.8.0 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/CMP-2101

Link to docs preview (VPN required):
[Security Profiles Operator 0.8.0](https://file.rdu.redhat.com/antaylor/CMP-2101/security/security_profiles_operator/spo-release-notes.html#spo-release-notes-0-8-0)

QE review:
- [x] QE has approved this change.

Additional information:
The errata link provided in this PR **will not work** until the errata is published. This PR will be merged after the release is final.

### CODE REVIEW

**Review by Vincent056 — APPROVED** [review]
/lgtm

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,18 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-8-0"]
+== Security Profiles Operator 0.8.0
+
+The following advisory is available for the Security Profiles Operator 0.8.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:4689[RHBA-2023:4689 - OpenShift Security Profiles Operator bug fix update]
```
Comment (kowen-rh) [inline security/security_profiles_operator/spo-release-notes.adoc:20]: This link is giving me a 404. Is this published yet?

**Review by kowen-rh — COMMENTED** [review]
The link to the bug fix update is giving me a 404, but otherwise, the text looks good to me! :+1:

/remove-label peer-review-in-progress
/label peer-review-done

File: security/security_profiles_operator/spo-release-notes.adoc (hunk 1 above)
Comment (sheriff-rh) [reply security/security_profiles_operator/spo-release-notes.adoc]: The content was not live at the time of your review, however I had an RHSA in the link, so after publication it 404'ed for me too :) 

I fixed it to the correct RHBA tag and all the content works now. This is why I always test prior to merge!

**Comment by sheriff-rh** [comment]
Content is now shipped live. Merging.

**Pull request merged**